### PR TITLE
Add fontPens to extras

### PR DESCRIPTION
**Pull request by benkiel**

### CODE REVIEW

**Comment by benkiel** [comment]
@anthrotype like so?

**Comment by anthrotype** [comment]
I think so. Can you do

pip install -e .[pens]

and get fontPens as well as defcon and the other deps?

**Comment by benkiel** [comment]
@anthrotype yes, that works!

File: setup.py [hunk 1]
```diff
@@ -32,6 +32,9 @@
         "fonttools>=3.1.2",
         "ufoLib>=2.1.0",
     ],
+    extras_require={
+        'pens': ["fontPens>=0.1.0", ],
```
Comment (anthrotype) [inline setup.py]: you could make this list multi-line if you think more could be added later (or just remove the final comma for now)

**Comment by anthrotype** [comment]
(this `pip install defcon[pens]` extras should probably also be documented somewhere I guess)

**Comment by benkiel** [comment]
@anthrotype changed setup.py and added to the documentation.

**Comment by benkiel** [comment]
@anthrotype can you merge this, I don't have commit rights to defcon. Thank you!

**Comment by anthrotype** [comment]
>  I don't have commit rights to defcon

ping @typesupply ?

**Pull request merged**